### PR TITLE
Minor refactor of acid spray crossed

### PR DESCRIPTION
**Pull request by spookydonut**
## About The Pull Request
god this is still pretty bad but at least its not directly calling crossed()

### CODE REVIEW

File: code/game/objects/effects/aliens.dm [hunk 1]
```diff
@@ -59,31 +59,32 @@
...
+	if(HAS_TRAIT(src, TRAIT_FLOORED))
+		take_overall_damage(0, rand(12, 14), run_armor_check("chest", "acid"))
+		UPDATEHEALTH(src)
+		to_chat(src, "<span class='danger'>You are scalded by the burning acid!</span>")
+		return
+	to_chat(src, "<span class='danger'>Your feet scald and burn! Argh!</span>")
+	if(!(species.species_flags & NO_PAIN))
+		emote("pain")
+	next_move_slowdown += slow_amt
+	var/datum/limb/affecting = get_limb("l_foot")
```
Comment (Hughgent) [inline code/game/objects/effects/aliens.dm:78]: for these `get_limb()` please use the associated defines. `BODY_ZONE_PRECISE_L_FOOT` in this example

**Review by Hughgent — CHANGES_REQUESTED** [review]

**Comment by Hughgent** [comment]
Please also use the PR template

**Comment by spookydonut** [comment]
> 
> 
> Please also use the PR template

Which part should I be using?

**Comment by Hughgent** [comment]
Essentially all of it.

**Review by Hughgent — APPROVED** [review]

**Review by TiviPlus — APPROVED** [review]
Non player facing changes dont necessarily need a CL hugh

**Comment by spookydonut** [comment]
@Hughgent just merge it, dont need TM

**Pull request merged**